### PR TITLE
Removed resizeMode from HomeScreen images

### PR DESCRIPTION
**Pull request by NickLiggett**
## What does this PR do?
- [] New feature (adds functionality)
- [x] Debugging/Refactor
 
### Where should your reviewer start?
- `HomeScreen.js`
### What (if any) features are you implementing?
-
### What (if anything) did you refactor?
- Removed resizeMode from HomeScreen images.
### What issues does this PR close?
-
### Is there anything that you need from your teammates?
-
### Any other comments, questions, or concerns?
-
### How Has This Been Tested?
- [x] I tested my changes in the browser
- [] I tested with Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new bugs
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/GameDetailsScreen.js [hunk 1]
```diff
@@ -14,11 +13,15 @@ const GameDetailsScreen = ({ game, myGames, addGame, removeGame, setModalVisible
   const { title, image, genres, consoles } = game;
   
   useEffect(() => {
-    if (myGames.find((element) => element.game_title === game.game_title) !== undefined) {
+    console.log(game)
```
Comment (tfettig22) [inline src/components/GameDetailsScreen.js]:
```suggestion

```

File: src/components/GameDetailsScreen.js [hunk 2]
```diff
@@ -14,11 +13,15 @@ const GameDetailsScreen = ({ game, myGames, addGame, removeGame, setModalVisible
...
+    console.log(myGames)
```
Comment (tfettig22) [inline src/components/GameDetailsScreen.js]:
```suggestion

```

File: src/components/GameDetailsScreen.js [hunk 3]
```diff
@@ -14,11 +13,15 @@ const GameDetailsScreen = ({ game, myGames, addGame, removeGame, setModalVisible
...
+    if (myGames.find((element) => element.game_title === game.title) !== undefined) {
       setHasGame(true);
     }
   }, []);
 
+  console.log(hasGame)
```
Comment (tfettig22) [inline src/components/GameDetailsScreen.js]:
```suggestion

```

**Review by aspitz1 — APPROVED** [review]
Thanks Nick!

**Review by tfettig22 — APPROVED** [review]

**Pull request merged**